core/node/da_dispatcher/src/da_dispatcher.rs

```diff
@@ -120,99 +127,138 @@ impl DataAvailabilityDispatcher {
         let client = self.client.clone();
         let request_semaphore = self.request_semaphore.clone();
         let notifier = Arc::new(Notify::new());
-        let pending_blobs_sender = tokio::spawn(async move {
-            let mut spawned_requests = vec![];
+        pending_blobs.spawn(async move {
```

Comment:
Same as the other comment, but clarifying that this task is in charge **send**ing blobs to the dispatcher.